sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/package-info.java

```diff
@@ -3,7 +3,7 @@
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
 /**
- * Package containing the data models for KeyVaultAccessControlClient and KeyVaultBackupClient. The key vault client
- * performs cryptographic key operations and vault operations against the Key Vault service.
+ * Package containing the data models for KeyVaultBackupClient. The key vault client performs cryptographic key
```

Comment:
This got changed by Autorest, is there any way of replacing the string with what we used to have?

Reply:
We can manually revert this change, but if this is something coming from swagger then will be hard to change.
@srnagar

Reply:
This will then contain the client name for the library we used Autorest last with.